### DISCUSSION TITLE
Is there a way to convert an Insomnia spec to an Open API Spec?

### DISCUSSION THREAD

**Discussion by kpalania** [Kong/insomnia#4337] [Help] [answered]

**Comment by wdawson** [comment, marked as answer]
Hi @kpalania , we're working on removing Insomnia-specific metadata from the spec files we store in your git repositories so you won't have to worry about this in the future. But for now, you can use the `inso` CLI and specifically the `inso export spec` command, which is [documented here](https://docs.insomnia.rest/inso-cli/cli-command-reference/inso-export-spec).

I'll also be moving this to a GitHub discussion for better tracking :)
